langchain4j/src/main/java/dev/langchain4j/data/document/loader/AwsCredentials.java

```diff
@@ -0,0 +1,45 @@
+package dev.langchain4j.data.document.loader;
+
+import static dev.langchain4j.internal.Utils.areNotNullOrBlank;
+
+/**
+ * Represents an AWS credentials object, including access key ID, secret access key, and optional session token.
+ */
+public class AwsCredentials {
+
+    private final String accessKeyId;
+    private final String secretAccessKey;
+    private String sessionToken;
+
+    public AwsCredentials(String accessKeyId, String secretAccessKey, String sessionToken) {
+        this.accessKeyId = accessKeyId;
+        this.secretAccessKey = secretAccessKey;
+        this.sessionToken = sessionToken;
+    }
+
+    public AwsCredentials(String accessKeyId, String secretAccessKey) {
+        this.accessKeyId = accessKeyId;
+        this.secretAccessKey = secretAccessKey;
+    }
+
+
+    public String getAccessKeyId() {
```

Comment:
minor: please avoid using "get" in getters, we are trying to follow a modern record-style API.
`getAccessKeyId()` -> `accessKeyId()`

Reply:
done, updated the code to remove get prefix